tools/mavlogdump.py

```diff
@@ -235,6 +235,9 @@ def match_type(mtype, patterns):
         if (isbin or islog) and m_type == "FMT":
             output.write(m.get_msgbuf())
             continue
+        if (isbin or islog) and m_type == "FMTU":
```

Comment:
I think you're supposed to pass in `--meta` to get this, `MULT` and `UNIT`, all of which are required for sensible display...

I reckon we probably should include those up here, however....
```suggestion
        if (isbin or islog) and m_type in ["FMTU", "MULT", "UNIT"]:
```

Reply:
Nice, good catch